updateParticipantData.json

```diff
@@ -535,6 +535,23 @@
 		"dataType": "object",
 		"mustExist": false
 	},
+	"637153953[].844209241": {
```

Comment:
Why using `[]` after `637153953`?

Reply:
It's for validation of data in a variable length array. The `[]` part enforces the array data structure for validation rules so we can validate any index. Then, during validation, we remove the actual index for an equal comparison.
 
Rules format (JSON): https://github.com/episphere/connectFaas/blob/fa33180d31b0d1249502b295787ca26c80b380b6/updateParticipantData.json#L520
 
Validation (stripping the index to enforce the array structure): https://github.com/episphere/connectFaas/blob/fa33180d31b0d1249502b295787ca26c80b380b6/utils/sites.js#L399

Reply:
I see it now.